services/mail.js

```diff
@@ -11,6 +12,30 @@ async function sendTestEmailToMyself (myPersonalEmail) {
   return mailClient.send(msg)
 }
 
+async function sendNewUserEmail (data) {
+  const msg = {
+    to: data.email,
+    from: process.env.EMAIL,
```

Comment:
[nit] `process.env.EMAIL` should be more descriptive (maybe `EMAIL_SENDER_ADDRESS` or something)